scripts/quota_log.py

```diff
@@ -71,25 +90,54 @@ def save_results(disk_quota_data, cpu_hours_data, db_config):
             merged_results[project] = cpu_hours_data[project]
 
 
+    logging.info("Connecting to the Database: " + db_config['url'])
     # create db instance
     server = "http://{username}:{password}@{url}:{port}".format(
         url=db_config['url'],
         username=db_config['username'],
         password=db_config['password'],
         port=db_config['port'])
+    try:
+        couch = couchdb.Server(server)
+    except Exception, e:
+        logging.error(e.message)
+        exit()
 
-    couch = couchdb.Server(server)
     db = couch['uppmax']
+    logging.info('Connection established')
 
     # save results in the database
     for project in merged_results.values():
-        db.save(project)
+        try:
+            project_id, project_rev = db.save(project)
+        except Exception, e:
+            logging.error(e.message)
+            exit()
+        else:
+            logging.info('Project:' +  project_id, project['project'] + ' has been updated')
 
 
 
 if __name__ == "__main__":
-    with open('/home/funk_001/.ngi_config/statusdb.yaml', 'r') as config_file:
-        config = yaml.load(config_file.readall())
+
+    # parse arguments from command line
+    parser = argparse.ArgumentParser(description="Formats uquota \
+        information as a dict, and sends it to a given CouchDB.")
+
+    parser.add_argument("--config", dest="config", default=DEFAULT_CONFIG, \
+        help="Path to the DB config file.")
+
+    parser.add_argument("--log-level", dest="log_level", default="error", help="Log level: debug, info, error, critical")
```

Comment:
@Galithil - to complete the logging tutorial, would you recommend that this script logs to a file somewhere? Or a command line option to log to a file somewhere?